src/main/java/com/google/api/codegen/config/DiscoGapicInterfaceConfig.java

```diff
@@ -70,7 +71,7 @@ static DiscoGapicInterfaceConfig createInterfaceConfig(
       ResourceNameMessageConfigs messageConfigs,
       ImmutableMap<String, ResourceNameConfig> resourceNameConfigs) {
 
-    ImmutableMap<String, ImmutableSet<String>> retryCodesDefinition =
+    ImmutableMap<String, List<String>> retryCodesDefinition =
```

Comment:
I think this kind of defeats the purpose of having an ImmutableMap, because the list can be mutated underneath. What's the motivation for this?

Reply:
Ok, undid all the diffs away from ImmutableSet.